.credo.exs

```diff
@@ -0,0 +1,210 @@
+# This file contains the configuration for Credo and you are probably reading
```

Comment:
I've exported credo config so we can disable some checks if we consider that is not required for this project 👍